targets/utils.py

```diff
@@ -1,27 +1,15 @@
-import pprint
-
+#!/usr/bin/env python3
 
-def period_ns(freq):
-    return 1e9/freq
+import difflib
+import pprint
 
 
-def csr_map_update(csr_map, csr_peripherals):
-    csr_map.update(dict((n, v)
-        for v, n in enumerate(csr_peripherals, start=(max(csr_map.values()) + 1) if csr_map else 0)))
+def dict_set_max(d, key, value):
```

Comment:
This won't work if `key` doesn't exist in the dictionary.

Reply:
I'll fix this.